src/redis-cli.c

```diff
@@ -3860,8 +3878,8 @@ static void clusterManagerShowClusterInfo(void) {
                 return;
             };
             if (reply != NULL) freeReplyObject(reply);
-            printf("%s:%d (%s...) -> %d keys | %d slots | %d slaves.\n",
-                   node->ip, node->port, name, dbsize,
+            printf("%s:%d@%d (%s...) -> %d keys | %d slots | %d slaves.\n",
+                   node->ip, node->port, node->bus_port, name, dbsize,
```

Comment:
Does anyone rely on the exact output here? Then, this might be a change worth mentioning in release notes.

Reply:
should i remove this change?

Reply:
I think it's fine to keep.